roles/minio/templates/minio_rclone_secret.yaml.j2

```diff
@@ -2,11 +2,14 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: "{{ minio_rclone_secret_name }}"
+  labels:
+    secret-type: bucket
+    seldon-deploy: "true"
 type: Opaque
 stringData:
-  RCLONE_CONFIG_S3_TYPE: s3
-  RCLONE_CONFIG_S3_PROVIDER: minio
-  RCLONE_CONFIG_S3_ENV_AUTH: "false"
-  RCLONE_CONFIG_S3_ACCESS_KEY_ID: "{{ minio_access_key }}"
-  RCLONE_CONFIG_S3_SECRET_ACCESS_KEY: "{{ minio_secret_key }}"
-  RCLONE_CONFIG_S3_ENDPOINT: "http://minio.{{ minio_namespace }}.svc.cluster.local:9000"
+  RCLONE_CONFIG_MINIO_TYPE: s3
```

Comment:
Could we make the name of the endpoint customisable with the vars? 
Even if it defaults to MINIO would be nice to have flexibility.

Reply:
as in `RCLONE_CONFIG_{minio_remote}_TYPE`?

Reply:
If not too verbose `{{ minio_rclone_remote_name }}`

Reply:
But yes, something like that would be create. No strong opinion on exact variable name

Reply:
Done 👍🏻

Reply:
Will make sure to test this